proto/api.proto

```diff
@@ -148,8 +148,8 @@ message GetResponse {
 }
 
 // A PutRequest is arguments to the Put() method. Note that to write
-// an empty value, the value parameter is still specified, but both
-// Bytes and Integer are set to nil.
+// an empty value, the value parameter is still specified, but Bytes
```

Comment:
`value` is not nullable, so it's unclear what "specified" means here (maybe detritus from when it was nullable?). I'm not sure we need this comment any more; an empty `Value` behaves as one would expect.